spec/index.html

```diff
@@ -877,7 +901,7 @@ <h3>Blank Nodes</h3>
     <h3>Triple Terms</h3>
 
     <p>An [=RDF triple=] used as the [=object=] of another [=triple=] is called a <dfn class=export >triple term</dfn>.
-      In a given [=RDF graph=], a [=triple=] can appear as a [=triple term=], an [=asserted triple=], or both.
+      In a given [=RDF graph=], a [=triple=] can appear as a [=triple term=], as an [=asserted triple=], or both.
```

Comment:
There should either be one `as` that leads into the list (and no `as` within any list item), _or_ have have one `as` within each list item. I previously requested the former (which I still prefer), and that is what is being partially edited in this PR; so now I request the latter, based on the edit already in this PR. Perhaps folks could +1/-1 this change request.
```suggestion
      In a given [=RDF graph=], a [=triple=] can appear as a [=triple term=], as an [=asserted triple=], or as both.
```

Reply:
> There should either be one as that leads into the list (and no as within any list item), or have have one as within each list item. I previously requested the former (which I still prefer), ...

Okay. I'm fine with reverting my change here.
```suggestion
      In a given [=RDF graph=], a [=triple=] can appear as a [=triple term=], an [=asserted triple=], or both.
```